### PR TITLE
Simplify github templates

### PR DESCRIPTION
**Pull request by tobiasdiez**
I find the checkboxes `Changes in pull request outlined? (What, why, ...)` and `Tests green?` a bit redundant. Especially since the build result is displayed directly under the PR message. So with this PR the template is even more simplified.

- [x] Change in CHANGELOG.md described? - pointless
- [x] Changes in pull request outlined? (What, why, ...)
- [x] Tests created for changes? - not applicable
- [x] Tests green?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-* JabRef version (available in the About box):
-* Operating system and version:
 * Steps to reproduce:
   1. ...
   2. ...
   3. ...
-  * If applicable, excerpt of the bibliography file, screenshot, and excerpt of log (available in the error console)
+* If applicable, excerpt of the bibliography file, screenshot, and excerpt of log (available in the error console)
+* JabRef version (available in the About box):
```
Comment (koppor) [inline .github/ISSUE_TEMPLATE.md]: I liked to read the JabRef version first. Maybe we should reuse your idea from the PR template:

```
JabRef version <version as shown in the about box> on <Windows 10|Ubuntu 14.04|Mac OS X 10.8|...>
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
+<describe the changes you have made here>
```
Comment (koppor) [inline .github/PULL_REQUEST_TEMPLATE.md]: I like that. May be we should add `motivation, considered alternatives, ...`. Not sure if that is too much.

**Comment by matthiasgeiger** [comment]
The updated pull request template is fine for me. 

But in my opinion the previous Issue-Template was better ;-)

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (simonharrer) [inline .github/PULL_REQUEST_TEMPLATE.md]: In favor of also the why, because this is often forgotten ...

Comment (Siedlerchr) [inline .github/PULL_REQUEST_TEMPLATE.md]: Describe the changes is enough, maybe just add a simple:
Why did you make the changes?  -> The one could add here the issue ref or just describe a one liner.
I think motivation, considered alternatives are only usefuil for major architecture changes

**Comment by tobiasdiez** [comment]
@koppor when you change this, can you also add `Screenshots added (for bigger UI changes)` to the PR template? Thanks. :+1:

**Comment by stefan-kolb** [comment]
We should add a hint that if they are using a dev version of JabRef they should always first try the latest version before reporting a bug. While it is good that they use the dev version and try new stuff before it is released, in the last days it happened more often that they reported bugs that were already fixed in the most recent version.

**Comment by koppor** [comment]
Done.

**Comment by oscargus** [comment]
Wouldn't it make sense to ask the user to try the latest development version as well? Or at least look at the ChangeLog of the development version?

**Comment by stefan-kolb** [comment]
For full versions I think its ok to shift this responsibility on our side. For development versions a user should make sure it is broken on the latest dev build.

**Comment by tobiasdiez** [comment]
Thanks @koppor for the changes. I merge this in for now. Further modifications can be made in a new PR.

**Pull request merged**